README.yaml

```diff
@@ -94,133 +140,99 @@ usage: |2-
       name       = var.name
       stage      = var.stage
       delimiter  = var.delimiter
-      attributes = compact(concat(var.attributes, list("cluster")))
+      attributes = compact(concat(var.attributes, ["cluster"]))
```

Comment:
```suggestion
      attributes = ["cluster"]
```

Reply:
@aknysh Actually, because this is a root module example with `context.tf` or `context = module.this.context`, you do need to merge the attributes as shown in the example.

Reply:
@aknysh Actually, because this is a root module example with `context.tf` or `context = module.this.context`, you need to merge the attributes as shown in the example.